index.html

```diff
@@ -10,9 +10,10 @@
         <img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" alt="original trombones logo">
         <nav id="nav-bar">
             <a class="nav-link" href="#features">Features</a>
-            <a class="nav-link" href="#howitworks">How it works</a>
+            <a class="nav-link" href="#howitworks">How does it works</a>
             <a class="nav-link" href="#pricing">Pricing</a>  
```

Comment:
Please as I mentioned earlier respect html  semantic ex use ul and li

Reply:
- [ ] The previous reviewer said about considering giving the `why` the student should accept your changes, please be more specific in your comment, you need to provide all the necessary details to explain to the student why you think this change is valuable for him/her